ideas/310-core-team-proposal.md

```diff
@@ -78,38 +49,42 @@ For simplicity, the responsibilities are split into 2 categories:
 * **maintenance** - that means this effort is continuous and doesn't have any end date or exit criteria (publishing a mobile release is a good example, we don't stop)
 * **project** - something that has a clear exit criteria, with a few iterations, after which the task is considered completed (for instance, when LES option is there and works)
 
-### Core: Maintenance
+# Core: Maintenance
 
 #### 1. Battery usage
 
 #### 2. Application performance profiling
 
 #### 3. Mobile Releases
 
-> [pedro] Shouldn't all releases be considered here? Why only mobile? Feels like we'd losing a holistic view.
-> [igorm] not sure, the initial idea was to outscope desktop. Isn't it too much for one team?
-> [pedro] I'd defer to your opinion since you have the experience of handling the releases, but maybe not too much if we end up splitting in two teams. I was hoping that it would be a small increment to the workload to also handle the desktop release part.
+# Core: Projects - Active
 
-### Core: Projects
+#### Message Reliability Improvements
 
-> [dmitry] there is nothing about decentralization and discovery.
-> [igorm] can you suggest projects related to that?
-> [dmitry] establishing a network using desktop clients. we need atleast to reuse libp2p tools descibed here https://github.com/libp2p/go-libp2p/issues/375 . and another thing is discovery, geth plans to refactor discovery v5 and add discovery based on dns. we should integrate atleast this two things.
-> another point is censorship resitance. we need to ensure that we can update bootnodes if they get blocked in russia/china/other place.
-> [igorm] I will add those as 3 projects then
-> [igorm] done
+* Umbrella issue: https://github.com/status-im/status-react/issues/6757
+* Project GitHub tag: https://github.com/status-im/status-react/labels/chat-reliability
 
-#### 1. Message reliability improvements
+##### Project Vision
 
 The goal of this project is:
 * to make messaging extremely reliable;
 * to be able to prove that;
-* to be able to show that information to a user.
+* to be able to show that information to a user (especially it is important to
+    communicate if the message isn't delivered).
+
+##### Project Scope
+
+"Message reliability" in this project means:
+- being able to deliver a user's message;
+- or being able to communicate to the user that her message wasn't delivered;
+- with the current network topology.
```

Comment:
i would also add - topology _and level of decentralization_

Reply:
yeah, that is a very good addition!

Reply:
I've added this.